### PR TITLE
Rename executable. Clean up CMakeLists for testing.

### LINKED ISSUE #189: Clean up CI for using only one executable

### PR DESCRIPTION
**Pull request by uhetmaniuk**
Closes #189

### CODE REVIEW

**Comment by uhetmaniuk** [comment]
@rjones-sandia @nmm0 
After this pull request is approved, I will move the test folders around to gather them: contact, quasistatic, dynamic. 
I did not combine these changes with the new organization of folders as it would result in a large number of files changed (git considers files movement as a change).

**Comment by uhetmaniuk** [comment]
Once this PR is approved, I will ask if Julia can test the branch with her applications to verify that nothing is broken.

**Review by rjones-sandia — APPROVED** [review]
looks good/sorry for the delay

**Pull request merged**
